Source/damage.hpp

```diff
@@ -0,0 +1,99 @@
+#pragma once
+
+#include "engine/random.hpp"
+
+namespace devilution {
+
+struct Damage {

```

Comment:
I was going to suggest giving this struct a more generic name so it could be used for other pairs/ranges, however the only example I could find was min/max AC on ItemData... Might still be worth it even with the sole non-damage use?

Reply:
I think it should potetially _rely_ on a more generic range struct, not be replaced by it.

There is semantic value in having a type dedicated to represent damage and nothing else, as well as damange-specific methods.